src/lib/support/BufBound.h

```diff
@@ -37,7 +37,7 @@ namespace chip {
  */
 class BufBound
 {
-private:
+protected:
```

Comment:
nit: Instead of making all access available to subclasses, it might suffice to provide a `ResetBuffer(buffer, size)`.

Reply:
Done